### PR TITLE
Set PackageRequireLicenseAcceptance to true

### PR DESCRIPTION
**Pull request by Lxiamail**
to be consistent with try-convert tool

### CODE REVIEW

**Comment by Lxiamail** [comment]
@marklio We chatted about this change in the previous PR code review, but I fogot to apply before merging.

File: Directory.Build.props [hunk 1]
```diff
@@ -48,7 +48,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
     <PackageProjectUrl>https://github.com/Microsoft/dotnet-apiport</PackageProjectUrl>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
```
Comment (marklio) [inline Directory.Build.props:51]: Were you going to set it to true, or remove?

**Review by marklio — APPROVED** [review]

File: Directory.Build.props (hunk 1 above)
Comment (Lxiamail) [reply Directory.Build.props:51]: I trie to remove it, but the default value seems false. That's why I set it to true instead.

Comment (terrajobst) [inline Directory.Build.props:51]: FWIW, we mostly set it to false these days. It's just `true` happens to be the default, so we're not entirely consistent. I'd rather we change try-convert 😄

However, for global tools it doesn't really matter `dotnet tool install` seems to ignore it.

**Review by terrajobst — APPROVED** [review]

File: Directory.Build.props (hunk 1 above)
Comment (Lxiamail) [reply Directory.Build.props:51]: I see, in that case, I would rather set it back to false to avoid potential prompt from any system.

**Pull request merged**